### ISSUE TITLE
从 OneCycleLR 的参数来看，learing_rate 似乎一直都不变？

### ISSUE THREAD

**Issue by yt605155624** [yl4579/StarGANv2-VC#83]
mapping_network 一直是 2e-6，其他一直是 2e-4
因为 div_factor 和 final_div_factor 都是 1，那我理解是否无需设定 scheduler 了，直接所有的 learing_rate 都在 AdamW 里面写死为 2e-4 或者 2e-6 即可？

**Comment by yl4579** [comment]
The scheduler was copied directly from StarGANv2, including the learning rate of the mapping network being 1e-6. You can refer to https://github.com/clovaai/stargan-v2 for more details on the settings. I didn't change any hyperparameters here, so they were hardcoded.
